### PR TITLE
Add instructions for finding domain and app credentials

### PR DESCRIPTION
**Pull request by nbarbettini**

### CODE REVIEW

File: _source/_docs/api/getting_started/find_your_app_credentials.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+> Your client secret is missing. Replace {clientSecret} with the client secret of your Application. You can copy it from the Okta Developer Console in the details for the Application you created.
+
+The first step in building an application with Okta is creating an Okta Application resource that represents your project. Okta generates credentials for you (called a client ID and client secret) that you copy into your code to associate your project with the Okta Application.
+
+To find your Application's credentials, use the following steps:
+
+1. Sign in to your Okta organization with your administrator account.
+2. Click on Applications.
+3. If you haven't created an Okta Application to represent your project, click Add Application and follow the instructions. Otherwise, pick your Application in the list.
```
Comment (bdemers) [inline _source/_docs/api/getting_started/find_your_app_credentials.md]: "from" the list? vs "in" the list?
Feel free to ignore this :)

File: _source/_docs/api/getting_started/find_your_app_credentials.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+4. On the General tab, scroll down to the Client Credentials section.
+5. Copy the client ID and/or client secret using the Copy to Clipboard buttons to the right of each text field.
```
Comment (bdemers) [inline _source/_docs/api/getting_started/find_your_app_credentials.md]: On the blog we use **bold** for text that appears on the screen.  **client ID**,  I'm not sure if this qualifies here, but figured i'd note it

Comment (jakubvul) [reply _source/_docs/api/getting_started/find_your_app_credentials.md]: Did that.

File: _source/_docs/api/getting_started/find_your_app_credentials.md (hunk 1 above)
Comment (jakubvul) [reply _source/_docs/api/getting_started/find_your_app_credentials.md]: So free!

**Pull request merged**